fe/fe-core/src/main/java/com/starrocks/common/proc/FrontendsProcNode.java

```diff
@@ -46,7 +47,7 @@
     public static final ImmutableList<String> TITLE_NAMES = new ImmutableList.Builder<String>()
             .add("Name").add("IP").add("HostName").add("EditLogPort").add("HttpPort").add("QueryPort").add("RpcPort")
             .add("Role").add("IsMaster").add("ClusterId").add("Join").add("Alive")
-            .add("ReplayedJournalId").add("LastHeartbeat").add("IsHelper").add("ErrMsg")
+            .add("ReplayedJournalId").add("LastHeartbeat").add("StartTime").add("IsHelper").add("Version").add("ErrMsg")
```

Comment:
This will changed the index of ErrMsg. I think we should keep compatible with old version.
New added column should be placed at the end.

Reply:
Thanks for your comment! 
I will remember the compatibility issue later